catalogue/webapp/test/e2e/helpers/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { baseUrl } from './urls';
+
+export type cookieType = {
+  name: string;
+  value: string;
+  url?: string;
+  domain?: string;
+  path?: string;
+  expires?: number;
+  httpOnly?: boolean;
+  secure?: boolean;
+  sameSite?: 'Lax' | 'None' | 'Strict';
+};
+
+const prefixToggle = 'toggle_';
```

Comment:
```suggestion
const togglePrefix = 'toggle_';
```

We _tend_ towards value names that you would say, rather than trying to add a type of other metadata into the value name.